specification/cognitiveservices/data-plane/Face/v1.0/Face.json

```diff
@@ -2042,6 +1777,164 @@
         }
       }
     },
+    "HairProperties": {
+      "type": "object",
+      "description": "Properties describing hair attributes.",
+      "properties": {
+        "bald": {
+          "type": "number",
+          "description": "A number describing confidence level of whether the person is bald.",
+          "minimum": 0,
+          "maximum": 1
+        },
+        "invisible": {
+          "type": "boolean",
+          "description": "A boolean value describing whether the hair is visible in the image."
+        },
+        "hairColor": {
```

Comment:
>"hairColor": { [](start = 8, length = 14)

It is a better idea not to define objects inline because generated code could have naming issues.

Reply:
Solved